recipes/hypsoreader/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "HYPSO-Reading" %}{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DevAlvaroF/HYPSO-Reading/archive/v0.1.0.tar.gz
```

```suggestion
  url: https://github.com/DevAlvaroF/HYPSO-Reading/archive/v{{ version }}.tar.gz
```
Makes sure that the bot can update it later.